packages/react-components/src/components/StartCaptionsButton.tsx

```diff
@@ -84,14 +90,17 @@ export const _StartCaptionsButton = (props: _StartCaptionsButtonProps): JSX.Elem
   };
 
   const captionsOptions: _captionsOptions = {
-    spokenLanguage: currentSpokenLanguage ?? 'en-us'
+    spokenLanguage: currentSpokenLanguage === '' ? defaultSpokenLanguage : currentSpokenLanguage
   };
 
   const onToggleStartCaptions = (): void => {
     if (props.checked) {
       onStopCaptions();
     } else {
       onStartCaptions(captionsOptions);
```

Comment:
I guess we will need to await the onStartCaptions(captionsOptions);
Then we go ahead to onSetSpokenLanguage(captionsOptions.spokenLanguage);